apriltag:
  mean_x: -2.28
  mean_y: -48.47
  std_dev_x: 39.57
  std_dev_y: 49.77
  color:
    - 250
    - 0
    - 0   # RGB color for the ellipse, green in this case
  transparency: 0.5
  image_path: '/home/akorycki/oasis/data/benchmark_dataset2/images/1733182663.879160200.png'
  #image_path: '/home/akorycki/oasis/data/benchmark_dataset2/images/1733182663.679160097.png'
